org.doctales.ant-contrib.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "org.doctales.ant-contrib",
+    "description": "Utility plugin which provides the ant-contrib library.",
+    "keywords": ["Ant"],
+    "homepage": "https://github.com/doctales/org.doctales.ant-contrib/",
+    "vers": "1.0.beta3",
```

Comment:
The version number is not a valid SemVer version number. Not sure if code in DITA-OT is able to handle this. Need to check before merge.

Reply:
OK. I checked on semver.org how they declare betas and this is how they do it, see https://semver.org/spec/v1.0.0-beta.html. If I should rename the plugin to whatever, please let me know.

Reply:
That should not be valid semver. `1.0.0.beta3` is, but not `1.0.beta3` because the first three version tokens are numbers.